protocol/revapi.json

```diff
@@ -24,7 +24,7 @@
           "code": "java.method.addedToInterface",
           "new": {
             "matcher": "java-package",
-            "match": "/io\\.camunda\\.zeebe\\.protocol\\.record\\.value(\\..*)?/"
+            "match": "/io\\.camunda\\.zeebe\\.protocol\\.record(\\..*)?/"
```

Comment:
🔧 This changes what is matched, so we should adjust its justification.

Please also consider whether it is okay for the `io.camunda.zeebe.protocol.record.intent` package to ignore new interface methods. I think so because there are only 2 interfaces in that package, but please have a look for yourself.

Reply:
I understood every interface in the protocol in general is not meant to be implemented, from an external user's point of view. Did I misinterpret this?

Reply:
That's true, but the justification specifically mentions Protocol Record Value interfaces. 

>Ignore new methods for Protocol Record Value interfaces, ...

🔧 Let's adjust it to Protocol Record interfaces.

Reply:
Good catch :+1:

Reply:
I updated the justification :+1: